action.yaml

```diff
@@ -56,7 +59,7 @@ inputs:
   # Enrichers
   CF_ENRICHERS:
     required: false
-    description: "list of integrations separated by commas: jira, github"
+    description: "list of integrations separated by commas: Acceptable values are git, jira. ex: 'jira, git'"
```

Comment:
There are unnessecary repetitions (git, jira) in the updated test, maybe it's better to keep the old one